charts/agent-control/templates/installation-job.yaml

```diff
@@ -14,9 +16,30 @@ spec:
   template:
     spec:
       restartPolicy: Never
-      serviceAccountName: {{ include "newrelic.common.naming.truncateToDNSWithSuffix" (dict "name" .Release.Name "suffix" "flux-install-job") }}
+      serviceAccountName: {{ include "newrelic.common.naming.truncateToDNSWithSuffix" (dict "name" .Release.Name "suffix" "install-job") }}
+      {{- if .Values.agentControlCd.enabled }}
+      volumes:
+        - name: flux-values
+          secret:
+            secretName: {{ include "agent-control.secret.name" . }}-flux
+        {{- if .Values.agentControlCd.repositoryCertificateSecretReferenceName }}
+        - name: flux-cert-secrets
+          secret:
+            secretName: {{ .Values.agentControlCd.repositoryCertificateSecretReferenceName }}
+            items:
+              - key: ca.crt
+                path: ca.crt
+              - key: tls.crt
+                path: tls.crt
+              - key: tls.key
+                path: tls.key
+        {{- end }}
+        {{- with .Values.installation.extraVolumes }}
+        {{- toYaml . | nindent 8 }}
+        {{- end }}
+      {{- end }}
       containers:
-        - name: install-flux
+        - name: install-agent-control-deployment
```

Comment:
Nit: Change the name to better express what it does.

The name is misleading. This container installs the "deployment" and "flux". Maybe `install-agent-control` is enough?

Reply:
You are 100% right